pkg/api/api.go

```diff
@@ -54,6 +54,8 @@ const (
 	CreateOAuthClientManagedClusterActionName = "console-create-oauth-client"
 	OAuthServerCertManagedClusterViewName     = "console-oauth-server-cert"
 
+	PluginI18nAnnotation = "console.openshift.io/default-i18next-namespace"
```

Comment:
I should have made this comment on the enhancement, but I feel like this annotation sounds like the namespace name is meant to be the value. Maybe `console.openshift.io/use-default-i18next-namespace`? Or maybe even `console.openshift.io/use-i18next` to keep it short?

Reply:
love the short one 👍